src/main/webapp/my-lists.html

```diff
@@ -0,0 +1,137 @@
+        <link rel="stylesheet" type="text/css" href="general.css">
+        <link rel="stylesheet" type="text/css" href="my-lists.css">
+    </head>
+    <body onload="displayLoginLogoutLink(), getLists()">
+        <nav class="navbar navbar-default navbar-fixed-top" id="top">
+            <div class="container">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="/homepage.html">Sullivan</a>
+                </div>
+                <div class="collapse navbar-collapse">
+                    <ul class="nav navbar-nav navbar-right">
+                        <li><a href="#about">ABOUT</a></li>
+                        <li><a href="#about">CONTACT</a></li>
+                        <li id="account-dropdown" class="dropdown"><a class="dropdown-toggle" data-toggle="dropdown" href="#">MY ACCOUNT <span class="caret"></span></a>
+                            <ul class="dropdown-menu">
+                                <li><a id="profile">Profile</a></li>
+                                <li><a id="history">History</a></li>
+                                <li><a id="availability-settings">Manage Availability</a></li>
+                                <li><a id="my-students">My Students</a></li>
+                                <li><a id="my-lists">My Lists</a></li>
+                                <li><a id="my-progress">My Progress</a></li>
+                                <li><a id="tutor-session-settings">Manage Tutoring Sessions</a></li>
+                            </ul>
+                        </li>
+                        <li id="login"><a id="login-url"><span class="glyphicon glyphicon-user"></span> Login</a></li>
+                        <li id="logout"><a id="logout-url"><span class="glyphicon glyphicon-log-in"></span> Logout</a></li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <div class="container-fluid bg-white" id="lists">
+            <h1>My Lists</h1>
+            <p>Here, you can create your own book "playlists" geared towards a particular topic that you are familiar with.
+                These lists are public and can help students narrow down the best resources available for the topic they are interested in learning more about.</p>
+            <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#new-list-modal" style="display:block">
+                + New List
+            </button>
+            <div class="list-group col-md-4" id="list-names" style="margin-top:3%"></div>
+            <div id="list-info-container" style="margin-top:3%"></div>
+        </div>
+
+        <!-- Modal -->
```

Comment:
What is a modal?

Reply:
It's like a pop up. There's examples here: https://getbootstrap.com/docs/4.0/components/modal/